guides/07_FAQs.md

```diff
@@ -78,6 +78,11 @@
     - [How can I speed up my meeting join times?](#how-can-i-speed-up-my-meeting-join-times)
   - [Messaging](#messaging)
     - [How do I receive Amazon Chime SDK channel messages without using the Amazon Chime SDK for JavaScript?](#how-do-i-receive-amazon-chime-sdk-channel-messages-without-using-the-amazon-chime-sdk-for-javascript)
+  - [Mobile SDK](#mobile-sdk)
+    - [What does each MeetingSessionStatusCode mean?](#what-does-each-meetingsessionstatuscode-mean)
```

Comment:
nit: Lets backtick the coding naming constructs. For example: `MeetingSessionStatusCode` here. I see you are using it in some places but suggest to do this overall for consistency.